wiki/Mapping_Techniques/Basics/en.md

```diff
@@ -1,25 +1,150 @@
-Flow of beatmap design
-----------------------
+# Basics
```

Comment:
The title will look like this <https://new.ppy.sh/wiki/People/Beatmap_Nomination_Group/>
(note that `People` is still here)